api/pkg/transformer/feast/feature_table.go

```diff
@@ -0,0 +1,98 @@
+package feast
```

Comment:
Can you add documentation to all the functions introduced within `feature_table.go` .